packages/apps/composer-app/src/main.tsx

```diff
@@ -164,6 +161,8 @@ const main = async () => {
       [MarkdownMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-markdown')),
       [MermaidMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-mermaid')),
       [MetadataMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-metadata')),
+      ...(isSocket ? { [PwaMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-pwa')) } : {}),
+      [NativeMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-native')),
```

Comment:
can you `isSocket` this one too? I think if it's in list of available plugin it will show up in the settings as available to enable